src/overlays/actors/ovl_En_Mu/z_en_mu.c

```diff
@@ -159,12 +165,12 @@ void EnMu_Update(Actor* thisx, GlobalContext* globalCtx) {
 
     this->collider.dim.pos = pos;
 
-    CollisionCheck_SetOC(globalCtx, &globalCtx->colChkCtx, &this->collider);
+    CollisionCheck_SetOC(globalCtx, &globalCtx->colChkCtx, &this->collider.base);
     SkelAnime_Update(&this->skelAnime);
     Actor_UpdateBgCheckInfo(globalCtx, &this->actor, 0.0f, 0.0f, 0.0f, 4);
     this->actionFunc(this, globalCtx);
     talkDist = this->collider.dim.radius + 30.0f;
-    func_800343CC(globalCtx, this, &this->dialogState, talkDist, EnMu_GetFaceReaction, EnMu_CheckDialogState);
+    func_800343CC(globalCtx, thisx, &this->dialogState.unk_00, talkDist, EnMu_GetFaceReaction, EnMu_CheckDialogState);
```

Comment:
```suggestion
    func_800343CC(globalCtx, &this->actor, &this->dialogState.unk_00, talkDist, EnMu_GetFaceReaction, EnMu_CheckDialogState);
```
If this doesn't work than change `this->actor` to `thisx`